src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/CodeGenerator.cs

```diff
@@ -170,18 +221,52 @@ public void GenerateOutput()
             consumeModelCSFileContent = Utils.FormatCode(consumeModelCSFileContent);
             var modelProjectFileContent = GenerateModelProjectFileContent(includeLightGbmPackage,
                 includeMklComponentsPackage, includeFastTreePackage, includeImageTransformerPackage,
-                includeImageClassificationPackage, includeRecommenderPackage,
+                includeImageClassificationPackage, includeRecommenderPackage, includeOnnxModel,
                 _settings.StablePackageVersion, _settings.UnstablePackageVersion);
 
             return (modelInputCSFileContent, modelOutputCSFileContent, consumeModelCSFileContent, modelProjectFileContent);
         }
 
+        internal (string ModelInputCSFileContent, string ModelOutputCSFileContent, string ConsumeModelCSFileContent,
+    string ModelProjectFileContent) GenerateAzureAttachImageModelProjectContents(string namespaceValue)
+        {
+            var classLabels = GenerateClassLabels();
+
+            // generate ModelInput.cs
+            var modelInputCSFileContent = GenerateModelInputCSFileContent(namespaceValue, classLabels);
+            modelInputCSFileContent = Utils.FormatCode(modelInputCSFileContent);
+
+            // generate ModelOutput.cs
+            var modelOutputCSFileContent = new OnnxModelOutputClass()
+            {
+                Namespace = namespaceValue,
+                Target = _settings.Target,
+            }.TransformText();
+            modelOutputCSFileContent = Utils.FormatCode(modelOutputCSFileContent);
+
+            // generate ConsumeModel.cs
+            var consumeModelCSFileContent = new AzureAttachImageConsumeModel()
+            {
+                Namespace = namespaceValue,
+                Target = _settings.Target,
+            }.TransformText();
+            consumeModelCSFileContent = Utils.FormatCode(consumeModelCSFileContent);
+
+            // generate Model.csproj
```

Comment:
Local code format starts comments with a capital letter. Would be good to search for "// [a-z]" and replace the first char with upper case.

```suggestion
            // Generate Model.csproj
```

Reply:
## Resolved